### PR TITLE
Many small fixes (part1)

### PR DESCRIPTION
**Pull request by andrixnet**
I've made some small changes in styles.
Corrected some romanian strings.
Corrected some improper strings in all translation files.

@kojoty please verify PL translations
@harrieklomp please verify NL translations

Please apply https://github.com/opencaching/opencaching-pl/compare/master...andrixnet:master#diff-6f67fb0fbae06362dc74e224ffb048df this. It removes an unnecessary line in `waypoint_type` table which now is handled much more elegantly.

### CODE REVIEW

File: lib/languages/pl.php [hunk 1]
```diff
@@ -694,6 +695,7 @@
     'coordinates_wp' => 'Współrzędne',
     'describe_wp' => 'Opis',
     'add_new_waypoint' => 'Dodaj nowy waypoint',
+    'choose_waypoint_type' => 'Wybierz typ punktu trasy',
```
Comment (mzylowski) [inline lib/languages/pl.php]: "Wybierz typ waypointa", sounds better.
We don't translate 'waypoint' word on PL node.

Fix should be done by amending 6ca1f6a2ded0edd138ba9148199433d32783fb8e commit.

File: lib/languages/pl.php [hunk 2]
```diff
@@ -2543,6 +2545,7 @@
     'N' => 'N',
     'C' => 'K',
     'Entry' => 'Wpis',
+    'Entry_latest' => 'Ostatnie wejście',
```
Comment (mzylowski) [inline lib/languages/pl.php]: "Ostatni wpis"

Fix should be done by amending 6ca1f6a commit.

File: sqlAlters/2017-03-22_waypoint_types_correction.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+-- 2017-03-22: Form select default displayed value
+-- @author: andrixnet
+
+-- this indicator value is no longer necessary
+
+DELETE FROM waypoint_type
+    WHERE id=-1;
```
Comment (mzylowski) [inline sqlAlters/2017-03-22_waypoint_types_correction.sql]: http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Comment (andrixnet) [inline sqlAlters/2017-03-22_waypoint_types_correction.sql]: My mistake. Oops.

**Comment by andrixnet** [comment]
Corrected PL strings as suggested. 
Integrated NL string corrections. 
Corrected newline problem. 
Cleaned up the commits. (maybe a bit too much, still git gives me a hard time). 
Added a new change for openchecker in viewcache. IMO centering it breaks the flow of the page in a bad way. I aligned it 20px like the rest of the UI controls.

**Comment by andrixnet** [comment]
Fixed https://github.com/opencaching/opencaching-pl/issues/923 entirely.

DB change applied at OCRO. 
Waiting for DB updated, then please merge.

**Comment by mzylowski** [comment]
OCPL DB is ready.
@harrieklomp after updating your DB. please merge this.

**Comment by harrieklomp** [comment]
DB updated on OC.NL
I am not sure how to merge this the correct way, so please merge it.

**Comment by deg-pl** [comment]
SQLalter done at OC UK

**Pull request merged**